tests/data/ReleaseApp/Api/Response/upgrade-analysis.json

```diff
@@ -0,0 +1,63 @@
+{
```

Comment:
File not in use.

Reply:
It used here  return $this->createHttpResponse(HttpUpgradeAnalysisHttpRequest::REQUEST_ENDPOINT);